src/x/config/config.go

```diff
@@ -42,37 +41,50 @@ var errNoFilesToLoad = errors.New("attempt to load config with no files")
 type Options struct {
 	DisableUnmarshalStrict bool
 	DisableValidate        bool
+
+	// LookupFunc -- see go.uber.org/config.Expand
+	Expand config.LookupFunc
```

Comment:
nit: remove the newline and comment (doesn't add any useful information that we can't get from the qualified type)

Reply:
Lol very true; I'm violating my own rule about non informative comments.